docs/userguide/usage_tracking.rst

```diff
@@ -1,82 +1,170 @@
 
-Parsl uses an **Opt-in** model to send usage statistics back to the Parsl development team to
-measure worldwide usage and improve reliability and usability. The usage statistics are used only for
-improvements and reporting. They are not shared in raw form outside of the Parsl team.
-
+Parsl uses an **Opt-in** model for usage tracking, allowing users to decide if they wish to participate. Usage statistics are crucial for improving software reliability and help focus development and maintenance efforts on the most used components of Parsl. The collected data is used solely for enhancements and reporting and is not shared in its raw form outside of the Parsl team.
 
 Why are we doing this?
 ----------------------
 
-The Parsl development team receives support from government funding agencies. For the team to continue to
-receive such funding, and for the agencies themselves to argue for funding, both the team and the agencies
-must be able to demonstrate that the scientific community is benefiting from these investments. To this end,
-it is important that we provide aggregate usage data about such things as the following:
+The Parsl development team relies on funding from government agencies. To sustain this funding and advocate for continued support, it is essential to show that the scientific community benefits from these investments.
+
+By opting in to share usage data, you actively support the ongoing development and maintenance of Parsl. (See:ref:`What is sent? <what-is-sent>` below).
+
+Opt-In Model
+------------
+
+We use an **opt-in model** for usage tracking to respect user privacy and provide full control over shared information. We hope that developers and researchers will choose to send us this information. The reason is that we need this data - it is a requirement for funding.
 
-* How many people use Parsl
-* Average job length
-* Parsl exit codes
+Choose the data you share with Usage Tracking Levels.
 
-By participating in this project, you help justify continuing support for the software on which you rely.
-(see :ref:`What is sent? <what-is-sent>` below).
+**Usage Tracking Levels:**
 
-Opt-In
-------
+* **Level 1:** Basic information such as Python version, Parsl version, and platform details.
+* **Level 2:** Configuration information including provider, executor, and launcher names.
+* **Level 3:** Workflow execution details, including the number of applications run, failures, and execution time.
 
-We have chosen opt-in collection rather than opt-out with the hope that developers and researchers
-will choose to send us this information. The reason is that we need this data - it is a requirement for funding.
+By enabling usage tracking, you support Parsl's development. 
 
-By opting-in, and allowing these statistics to be reported back, you are explicitly supporting the
-further development of Parsl.
+**To opt-in, set ``usage_tracking`` to the desired level (1, 2, or 3) in the configuration object (`parsl.config.Config`).**
```

Comment:
the backquote formatting doesn't work when i build this locally, inside the ** bold block. you could remove the bold block.